### PR TITLE
fix(lts/converge): Solve the problem that resources cannot be updated

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:

SUpports a diffsuppress function to avoid the resource changes during IDs set and compare.
The request body must be have the log group ID and the log stream ID if they are already created and needs be updated (coveraged).
image

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. supports new methods to calculate hash value.
2. adjust the structure building logic.
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST='./huaweicloud/services/acceptance/lts' TESTARGS='-run=TestAccLogConverge_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lts -v -run=TestAccLogConverge_basic -timeout 360m -parallel 4
=== RUN   TestAccLogConverge_basic
=== PAUSE TestAccLogConverge_basic
=== CONT  TestAccLogConverge_basic
--- PASS: TestAccLogConverge_basic (140.81s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lts       140.862s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: docs/resources/lts_log_converge.md [hunk 1]
```diff
@@ -77,6 +119,8 @@ The `log_mapping_config` block supports:
 
 * `target_log_group_id` - (Optional, String) Specifies the ID of the log group for target side.
 
+-> If you want to use an existing log group, this parameter (`target_log_group_id`) is required.
```
Comment (deer-hang) [inline docs/resources/lts_log_converge.md]:
```suggestion
  -> If you want to use an existing log group, this parameter (`target_log_group_id`) is required.
```

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_log_converge_test.go [hunk 2]
```diff
@@ -25,48 +25,60 @@ func TestAccLogConverge_basic(t *testing.T) {
...
-						acceptance.HW_LTS_LOG_CONVERGE_TARGET_LOG_GROUP_NAME),
-					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.target_log_group_id",
-						acceptance.HW_LTS_LOG_CONVERGE_TARGET_LOG_GROUP_ID),
+					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.target_log_group_name", name),
+					resource.TestCheckResourceAttrSet(rName, "log_mapping_config.0.target_log_group_id"),
+					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.log_stream_config.#", "0"),
+				),
+			},
+			{
+				Config: testLogConverge_basic_step2(name, 90),
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_log_converge_test.go]: It looks like the second parameter doesn't need to exist.
Same to function `testLogConverge_basic_step3`

**Comment by Lance52259** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/lts' TESTARGS='-run=TestAccLogConverge_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lts -v -run=TestAccLogConverge_basic -timeout 360m -parallel 4
=== RUN   TestAccLogConverge_basic
=== PAUSE TestAccLogConverge_basic
=== CONT  TestAccLogConverge_basic
--- PASS: TestAccLogConverge_basic (136.44s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lts       136.486s
```

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_log_converge_test.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_log_converge_test.go]: sure

File: docs/resources/lts_log_converge.md (hunk 1 above)
Comment (Lance52259) [reply docs/resources/lts_log_converge.md]: fixed.

**Pull request merged**